### PR TITLE
Rule rework

### PR DESCRIPTION
**Pull request by EmmaKnijn**

### CODE REVIEW

**Review by dimaguy — APPROVED** [review]

File: rules.md [hunk 1]
```diff
@@ -52,45 +52,40 @@ nav_order: 2
...
+**4.6** Use of profanity is okay, but disruptive use of profanity is not allowed.
 
-**4.9** Use of hate speech, racial slurs, threats and any other language that could be considered harmful to a person or a group of people is not allowed.
+**4.7** Use of hate speech, racial slurs, threats and any other language that could be considered harmful to a person or a group of people is not allowed.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.9.1** Discussion of such language, when it is clear that it is not being used for the intent of harm is okay, but should be undertaken with caution.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.7.1** Discussion of such language, when it is clear that it is not being used for the intent of harm is okay.
 
-**4.10** Explicit content of any form, including but not limited to: adult themes, erotic imagery, explicit innuendo, self harm and suicide, gore, shock content, etc. is not allowed.
+**4.8** Explicit content including: self harm and suicide, gore, shock content, etc. is not allowed.
```
Comment (Ale32bit) [inline rules.md:70]: NSFW (erotic content) should not be allowed.

File: rules.md [hunk 2]
```diff
@@ -52,45 +52,40 @@ nav_order: 2
...
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.10.3** Whenever anything makes you uncomfortable, please contact a staff member privately.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.8.3** Whenever anything makes you uncomfortable, please contact a staff member privately.
 
-**4.11** Stealing items or money from a player, shop, company or service, even if they have trusted you, even via an introspection module, is forbidden. This includes through abusing alt accounts, redistributing paid software, scamming, ransomware, impersonation.
+**4.9** Stealing items or money from a player, shop, company or service, even if they have trusted you, even via an introspection module, is forbidden. This includes through abusing alt accounts, scamming, impersonation.
 
-**4.12** Automatically taking excessive amounts of items from public sources, such as ender storages, is considered theft.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.9.1** Piracy and stealing through exploitation of bugs and other exploits in player software is allowed.
```
Comment (Ale32bit) [inline rules.md:80]: Ransomware and attacks with the purpose of maliciously harming the player should be forbidden.

File: rules.md [hunk 3]
```diff
@@ -52,45 +52,40 @@ nav_order: 2
...
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.10.3** Whenever anything makes you uncomfortable, please contact a staff member privately.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.8.3** Whenever anything makes you uncomfortable, please contact a staff member privately.
 
-**4.11** Stealing items or money from a player, shop, company or service, even if they have trusted you, even via an introspection module, is forbidden. This includes through abusing alt accounts, redistributing paid software, scamming, ransomware, impersonation.
+**4.9** Stealing items or money from a player, shop, company or service, even if they have trusted you, even via an introspection module, is forbidden. This includes through abusing alt accounts, scamming, impersonation.
 
-**4.12** Automatically taking excessive amounts of items from public sources, such as ender storages, is considered theft.
```
Comment (Ale32bit) [inline rules.md:81]: Why remove this rule?

**Review by Ale32bit — CHANGES_REQUESTED** [review]

File: rules.md (hunk 3 above)
Comment (EmmaKnijn) [reply rules.md:81]: It should help prevent rapid deflation of Kromer

File: rules.md (hunk 1 above)
Comment (Ale32bit) [reply rules.md:70]: Let's not forget there are underaged people playing the server, it is unethical to allow this.

Comment (dimaguy) [reply rules.md:70]: Whoa there, its not that I had a different position, Im not about to allow people to post nsfw content on channels, now discussing stuff reasonably like adults is something different

File: rules.md (hunk 2 above)
Comment (Ale32bit) [reply rules.md:80]: This is not Windows, the only way to get ransomwares and viruses in CC is by giving physical access to other people (or public) to the computer or disk drives, exploiting ad-hoc programs that make use of networking (especially wireless networking), the latter of which even a proper antivirus cannot detect easily.
It is fine to me to allow attacks as long as player data is not affected maliciously.

Comment (EmmaKnijn) [reply rules.md]: I updated it, do you think this is right?

File: rules.md (hunk 1 above)
Comment (EmmaKnijn) [reply rules.md]: Reverted

File: rules.md [hunk 4]
```diff
@@ -77,7 +77,7 @@ nav_order: 2
 
 **4.9** Stealing items or money from a player, shop, company or service, even if they have trusted you, even via an introspection module, is forbidden. This includes through abusing alt accounts, scamming, impersonation.
 
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.9.1** Piracy and stealing through exploitation of bugs and other exploits in player software is allowed.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**4.9.1** Piracy and stealing through exploitation of bugs and other exploits in player software is allowed, as long as player data isnt affected maliciously.
```
Comment (Ale32bit) [inline rules.md:80]: Add the missing apostrophe and lgtm

**Review by Ale32bit — CHANGES_REQUESTED** [review]

**Review by Ale32bit — APPROVED** [review]

**Pull request merged**